tests/test_atom_mapping.py

```diff
@@ -22,8 +22,9 @@ def test_mcs():
 def test_get_core_by_mcs():
     mol_a, mol_b, _ = get_hif2a_ligand_pair_single_topology()
     query = mcs(mol_a, mol_b).queryMol
-    core = get_core_by_mcs(mol_a, mol_b, query)
-    assert core.shape[1] == 2
+    for conformer_aware in [True, False]:
```

Comment:
nit: You can use pytest to run the test under different conditions
```
@pytest.mark.parametrize("conformer_aware", [True, False])
def test_get_core_by_mcs(conformer_aware):
    ...
```

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/898/commits/d1ff144ad06052249a89840280c8c4b5c9dbd176